content/docs/start/data-management/data-versioning.md

```diff
@@ -96,31 +96,22 @@ outs:
 
 You can upload DVC-tracked data or model files with `dvc push`, so they're
 safely stored [remotely](/doc/command-reference/remote). This also means they
-can be retrieved on other environments later with `dvc pull`. First, we need to
-set up a remote storage location:
-
-```cli
-$ dvc remote add -d storage s3://mybucket/dvcstore
```

Comment:
there was a reason for this ... we wanted this to be visible and about S3 (this makes it not reproducible for users who are trying to reproduce, but that was fine in this case)

Reply:
Makes sense. I made another iteration keeping the S3 remote in the same block as `dvc push` and then a `<details>` section underneath titled "That didn't work!". PTAL